src/main/java/nl/cherement/jak/entity/EventEntity.java

```diff
@@ -12,12 +15,15 @@
     public long id;
 
     @ManyToOne
+    @OnDelete( action = OnDeleteAction.CASCADE )
     public CardEntity card;
 
     @ManyToOne
```

Comment:
De naamgeving `from`  en `to` klinkt hier logisch. Maar verder gebruik van deze elementen word het erg verwarrend.
Plaats anders de naam `Column` bij.

Reply:
Dat is inderdaad misschien wel iets duidelijker ja. Maar kom je wel een beetje laat mee lol. Ik zal alles aanpassen hiervoor. Maar dan moeten de frontend en backend zo beiden gemerged om alles werkende te houden.
Als de frontend klaar is dan.